language/ar/ucp.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+*
+* @package phpBB Extension - Crizzo Max-width switch - Arabic
+* @copyright (c) 2014 Crizzo - www.crizzo.de
+* @license http://opensource.org/licenses/gpl-2.0.php GNU General Public License v2
+*
+ * Translated by Bassel Taha Alhitary (www.alhitary.net)
+*/
+
+if (!defined('IN_PHPBB'))
+{
+	exit;
+}
+
+if (empty($lang) || !is_array($lang))
+{
+	$lang = array();
+}
+
+$lang = array_merge($lang, array(
+	'MAXWIDTH'			=> 'تكبير عرض المنتدى ',
+	'MAXWIDTH_EXPLAIN'	=> 'اختيارك "نعم" يعني استخدام الحد الأقصى لعرض المنتدى ( التصفح بكامل الشاشة ).',
```

Comment:
Are those ( and ) correct? I think there should be a opening ( and a closing one ).